bitnami/fluent-bit/templates/_helpers.tpl

```diff
@@ -0,0 +1,59 @@
+{{/* vim: set filetype=mustache: */}}
+
+
+{{/*
+Return the proper fluent-bit image name
+*/}}
+{{- define "fluentBit.image" -}}
```

Comment:
In the prefix, I see more often kebab case rather than camel-case in our charts. 

```
fluent-bit.image
```

Could you change fluentBit with fluent-bit?

Reply:
Done 2087e1